src/pages/docs/[PageName].tsx

```diff
@@ -0,0 +1,41 @@
+import Layout from "../../components/Layout";
+import Markdown from '../../components/Markdown'
+import React from "react";
+import { NextPageContext } from "next";
+import { getTheme } from "../../utils/theme";
+import { HasTheme, HasMarkdown } from "../../interfaces";
+import { readAsString } from '../../utils/files';
+
+export default function Feedback({ theme, title, pageContents}: HasTheme & HasMarkdown) {
+
+    return <Layout title={title} theme={theme}>
+        <div className={`w-1/2 mx-auto my-4`}>
+            <Markdown markdown = {pageContents}/>
+        </div>
+    </Layout>;
+}
+
+const validPages: Record<string, string> = {
+    feedback: "Feedback",
+    terms: "Terms of Service",
+    privacy: "Privacy Policy",
+    conduct: "Code of Conduct",
+    disclosure: "Responsible Disclosure"
+};
+
+export async function getServerSideProps(context: NextPageContext) {
+
+    let {PageName = ""} = context.query;
+    let pageFile = PageName.toString();
+    let title = validPages[pageFile];
+
+    if (title) {
+        let theme = getTheme(context);
+        let pageContents = readAsString(`src/docs/${pageFile}.md`);
```

Comment:
Please use `path.join()` to create this string, it is more reliable across different operating systems

Reply:
readAsString calls path.join in the background which will normalize the output. Do you really want me to make it path.join(src, path.join(docs, pageFile.md)); ?